core/account.go

```diff
@@ -35,6 +35,20 @@ func (t *Textile) Decrypt(input []byte) ([]byte, error) {
 	return t.account.Decrypt(input)
 }
 
+// DeleteAccount removes the local account data by wiping the repo
+func (t *Textile) DeleteAccount(repoPath string) error {
+	if t.Started() {
+		// Node shouldn't be running when delete requested
+		return fmt.Errorf("node is started")
+	}
+	if 0 < len(t.CafeSessions().Items) {
```

Comment:
Is the existence of cafesessions enough of a check? or could a cafe be registered with no session that needs to be notified?

Reply:
cool, yep